[BUG] Babel-loader - hidden dependency

after running command `./node_modules/.bin/netlify-lambda build functions-source/`
I god
```
netlify-lambda: Building functions
Hash: 2f36233598c9262552e4
Version: webpack 4.20.2
Time: 1142ms
Built at: 2018-10-16 12:47:52
    Asset      Size  Chunks  Chunk Names
rating.js  1.21 KiB       0  rating
Entrypoint rating = rating.js
[0] ./rating.js 254 bytes {0} [built] [failed] [1 error]

ERROR in ./rating.js
Module build failed (from ../node_modules/babel-loader/lib/index.js):
TypeError: Cannot read property 'babel' of undefined
    at Object.module.exports (/Users/nikolay/Projects/autokwix/node_modules/babel-loader/lib/index.js:103:36)
```

after installing `babel-loader` - it started to work

how on earth?

@sw-yx Seems as if it's a mixed bag.

Ex. I'll get the `ERROR in Entry module not found: Error: Can't resolve 'babel-loader' in '/Users/[redacted]/Desktop/test/src/functions'` from a clean install but when I install the babel loader I get:
```
yarn run v1.13.0-20181004.1524
$ netlify-lambda serve src/lambda
netlify-lambda: Starting server
Lambda server is listening on 9000
Unhandled rejection Error: Cannot find module '/Users/[redacted]/Desktop/test/lambda/hello'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.resolve (internal/modules/cjs/helpers.js:30:19)
    at Object.clearCache (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/lib/serve.js:111:36)
    at /Users/[redacted]/Desktop/test/node_modules/netlify-lambda/bin/cmd.js:38:16
    at Array.forEach (<anonymous>)
    at Watching.handler (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/bin/cmd.js:37:32)
    at compiler.hooks.done.callAsync (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/node_modules/webpack/lib/Watching.js:98:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/[redacted]/Desktop/test/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/Users/[redacted]/Desktop/test/node_modules/tapable/lib/Hook.js:154:20)
    at Watching._done (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/node_modules/webpack/lib/Watching.js:97:28)
    at onCompiled (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/node_modules/webpack/lib/Watching.js:47:18)
    at hooks.afterCompile.callAsync.err (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/node_modules/webpack/lib/Compiler.js:552:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/[redacted]/Desktop/test/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/Users/[redacted]/Desktop/test/node_modules/tapable/lib/Hook.js:154:20)
    at compilation.seal.err (/Users/[redacted]/Desktop/test/node_modules/netlify-lambda/node_modules/webpack/lib/Compiler.js:549:30)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/[redacted]/Desktop/test/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:6:1)
```
which leads me to believe it has to do with version mismatch as from what I've gathered `netlify-lambda` brings its own version of `babel-loader`.

If I delete the `yarn.lock` file and `node_module` then redo a `yarn` to reinstall everything and run the `netlify-lambda serve src/lambda` it begins to work.

Im not as familiar with babel but I'll see what can be done to get it from messing up like so.

Another thing I've noticed is using Yarn Plug and Play it gives the same error with no `babel-loader` and no matter what I do it will not work at all even if I try installing it.

--- a/lib/build.js
+++ b/lib/build.js
@@ -37,13 +37,13 @@ function webpackConfig(dir, additionalConfig) {
   var babelOpts = { cacheDirectory: true };
   if (!haveBabelrc(dir)) {
     babelOpts.presets = [
-      ['@babel/preset-env', { targets: { node: getBabelTarget(envConfig) } }]
+      [require.resolve('@babel/preset-env'), { targets: { node: getBabelTarget(envConfig) } }]
     ];
 
     babelOpts.plugins = [
-      '@babel/plugin-proposal-class-properties',
-      '@babel/plugin-transform-object-assign',
-      '@babel/plugin-proposal-object-rest-spread'
+      require.resolve('@babel/plugin-proposal-class-properties'),
+      require.resolve('@babel/plugin-transform-object-assign'),
+      require.resolve('@babel/plugin-proposal-object-rest-spread')
     ];
   }
 
@@ -87,7 +87,7 @@ function webpackConfig(dir, additionalConfig) {
             `(node_modules|bower_components|${testFilePattern})`
           ),
           use: {
-            loader: 'babel-loader',
+            loader: require.resolve('babel-loader'),
             options: babelOpts
           }
         }

